cmd/metricshub/Makefile

```diff
@@ -6,8 +6,11 @@ ifndef API_KEY
 	$(error API_KEY is undefined)
 endif
 
+deploy-mainnet: check-env
+	gcloud run deploy metricshub-mainnet --source=${PWD}  --set-env-vars="GCP_PROJECT=textile-310716,BIGQUERY_DATASET=tableland_mainnet,BIGQUERY_TABLE=system_metrics,API_KEY=${API_KEY}" --region="us-west1" --allow-unauthenticated
+
 deploy-testnet: check-env
-	gcloud run deploy metricshub-testnet --source=${PWD}  --set-env-vars="GCP_PROJECT=textile-310716,BIGQUERY_DATASET=tableland_testnet,BIGQUERY_TABLE=system_metrics,API_KEY=${API_KEY}" --region="us-west1" --allow-unauthenticated
+	gcloud run deploy metricshub-testnet-v2 --source=${PWD}  --set-env-vars="GCP_PROJECT=textile-310716,BIGQUERY_DATASET=tableland_testnet_v2,BIGQUERY_TABLE=system_metrics,API_KEY=${API_KEY}" --region="us-west1" --allow-unauthenticated
```

Comment:
Whenever we deprecate the old validator and BQ dataset, we can do some renaming magic if we want some purity.